### DISCUSSION TITLE
Documentation Broken

### DISCUSSION THREAD

**Discussion by tom-dudman** [airjp73/rvf#327] [General]
The documentation appears to be broken.  On every page I get the following error:

```
Uncaught Error: Dynamic require of "react" is not supported
    at chunk-VVMP2TR3.js:1:383
    at ../../node_modules/use-sync-external-store/cjs/use-sync-external-store-shim/with-selector.production.min.js (chunk-ZH6XSU7G.js:1:17495)
    at chunk-ZH6XSU7G.js:1:15968
    at ../../node_modules/use-sync-external-store/shim/with-selector.js (chunk-ZH6XSU7G.js:1:18492)
    at chunk-ZH6XSU7G.js:1:15968
    at chunk-ZH6XSU7G.js:1:20608
```

This appears to be breaking Javascript, as I can't scroll the LHS menu without the page reloading, or view code samples.

Have tried multiple browsers & devices.  I think it's a recent change - it was working fine last week.

**Comment by airjp73** [comment]
Should be fixed now. Thanks for the heads up!

**Reply by tom-dudman** [reply]
Thanks!  Can confirm it's working for me.
